app/jobs/refresh_rss_feeds_job.rb

```diff
@@ -5,7 +5,7 @@ class RefreshRssFeedsJob < ApplicationJob
 
   def perform
     RssFeed.find_each do |feed|
-      RefreshRssFeedJob.perform_later(feed)
+      RefreshRssFeedJob.perform_later(feed) if feed.should_refresh?
```

Comment:
Doesn't a scope make more sense here? We're now loading all feeds to then ignore some

Reply:
Can you also add tests for this?